example/__main__.py

```diff
@@ -1,17 +1,16 @@
 from cmd import Cmd
 import objectbox
-import datetime
+from datetime import datetime
 from example.model import *
 
 
 # objectbox expects date timestamp in milliseconds since UNIX epoch
 def now_ms() -> int:
-    seconds: float = datetime.datetime.utcnow().timestamp()
-    return round(seconds * 1000)
+    return datetime.utcnow()
```

Comment:
Docs say this because of:
> Because naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. As such, the recommended way to create an object representing the current time in UTC is by calling datetime.now(timezone.utc).

Gonna change it :+1: